modules/oadp-configuring-client-burst-qps.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-client-burst-qps_{context}"]
+= Configuring the DPA with client burst and QPS settings
+
+The queries per second (QPS) setting determines the number of client requests or queries that are sent to the `velero` server. The burst setting determines how many requests from the client can be stored for processing.
```

Comment:
The burst setting determines how many requests will be sent before limits will be applied. Once the burst limit is hit, QPS determines how many additional requests will be sent per second.

Reply:
Done. thanks.